app/test_main.py

```diff
@@ -1,3 +1,69 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age",
+    [
+        pytest.param(None, 3),
+        pytest.param(3, "1"),
+        pytest.param(3, [3]),
+    ],
+    ids=[
+        "You put incorrect function argument",
+        "You put incorrect function argument",
+        "You put incorrect function argument",
+    ]
+)
+def test_for_correct_type_put_data(cat_age: int,
+                                   dog_age: int) -> None:
+    with pytest.raises(TypeError):
+        get_human_age(cat_age, dog_age)
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age",
+    [
+        pytest.param(0, 0, id="Function argument cannot be negative")
+    ]
+)
+def test_get_human_age_with_negative_values(cat_age: int,
+                                            dog_age: int) -> None:
+    assert cat_age >= 0 and dog_age >= 0
```

Comment:
You can make better test with actual using of negative values and add it to next decorator so there will be no need in using additional one